src/Containers/SlidingOutPanels.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import styled from 'styled-components';
+import { MainInterface, ResponsiveInterface } from '../Utils/BaseStyles';
+
+interface ImageProps {
+    key: number;
+    imageLink: string;
+    alt: string;
+}
+
+export interface SlidingOutPanelProps
+    extends MainInterface,
+        ResponsiveInterface {
+    images: ImageProps[];
+}
+
+export const SlidingOutPanels: React.FC<SlidingOutPanelProps> = ({
+    images,
+}): React.ReactElement => {
+    return (
+        <Items>
+            <Overlay>
+                <Image src={images[0].imageLink} alt={images[0].alt} />
+                <Image1 src={images[1].imageLink} alt={images[1].alt} />
+                <Image2 src={images[2].imageLink} alt={images[2].alt} />
+                <Image3 src={images[3].imageLink} alt={images[3].alt} />
```

Comment:
what if i have 5 images? use a map.

Reply:
this change has been implemented with a single component and custom styling from props